src/content_scripts/autocorrect.js

```diff
@@ -157,8 +163,8 @@ function countChars(str) {
 function deleteCaret(target, atext) {
     const count = countChars(atext);
     if (count > 0) {
-        const isSuccess = document.execCommand("delete", false);
-        if (isSuccess) {
+        // document.execCommand is deprecated, although there is not yet an alternative: https://developer.mozilla.org/en-US/docs/Web/API/Document/execCommand
+        if (document.execCommand("delete", false)) {
```

Comment:
Hmm, in general I don’t really like that (functionality in if or case or so), but in this case it may be okay. :slightly_smiling_face: